src/amp-story-player/amp-story-player-impl.js

```diff
@@ -1591,6 +1595,19 @@ export class AmpStoryPlayer {
     return index >= this.stories_.length || index < 0;
   }
 
+  /** @private */
+  initializeAutoplay_() {
+    if (!this.playerConfig_) {
+      return;
+    }
+
+    const {behavior} = this.playerConfig_;
+
+    if (behavior && behavior.autoplay === 'false') {
```

Comment:
This will break with
```
{
  "behavior": {
    "autoplay": false
  }
```

Reply:
Question for you or Enrique, is it even necessary to support the strings "true" and "false" here, given that the boolean forms can be used instead?

Reply:
Good call, forgot booleans are valid JSON hehe. PTAL.